gms/feature_service.hh

```diff
@@ -108,6 +108,7 @@ public:
     gms::feature tombstone_gc_options { *this, "TOMBSTONE_GC_OPTIONS"sv };
     gms::feature parallelized_aggregation { *this, "PARALLELIZED_AGGREGATION"sv };
     gms::feature keyspace_storage_options { *this, "KEYSPACE_STORAGE_OPTIONS"sv };
+    gms::feature full_parallelized_aggregation { *this, "FULL_PARALLELIZED_AGGREGATION"sv };
```

Comment:
It's better to be more specific, because I expect we'll refine it later. What will we call it then, EVEN_MORE_FULL_PARALLELIZED_AGGREGATION?

Also, can fold into the patch that uses it.

Reply:
Named it `UDA_NATIVE_PARALLELIZED_AGGREGATION`